parl/remote/remote_constants.py

```diff
@@ -21,3 +21,5 @@
 DESERIALIZE_EXCEPTION_TAG = b'[DESERIALIZE_EXCEPTION]'
 
 NORMAL_TAG = b'[NORMAL]'
+
+HEARTBEAT_INTERVAL_S = 10
```

Comment:
Can you add some comments about its unit? minute or second?

Reply:
done